src/top/mod.rs

```diff
@@ -12,18 +12,20 @@ use url::Url;
 #[command(rename_all = "kebab-case")]
 pub struct Opts {
     /// Interval to sample metrics at, in milliseconds
-    #[arg(default_value = "500", short = 'i', long)]
+    #[arg(default_value = "1000", short = 'i', long)]
```

Comment:
This avoids the default behavior where, because the metrics themselves are generated on a 1 second interval, one UI update has a throughput rate, and the next UI update just shows `--`, back and forth.

If users really want to change it, that's fine, but it doesn't actually change how often the metrics are updated over their respective GraphQL subscriptions... so it might actually be something we want to remove from letting users configure, since it has no functional bearing on the UI output.